glide.yaml

```diff
@@ -65,22 +34,97 @@ import:
 - package: k8s.io/kubernetes
   repo:    git@github.com:openshift/kubernetes
   version: release-1.9.0-beta.1
+
+# openshift second
 - package: github.com/openshift/api
-  version: release-3.9
+  version: master
 - package: github.com/openshift/client-go
-  version: release-3.9
+  version: master
+- package: github.com/openshift/imagebuilder
+  version: master
+
+# forks third
+# master
```

Comment:
not sure what "master" means here?

Reply:
i guess it means the team. i read it as a branch name.  might confuse others.

Reply:
> not sure what "master" means here?

sig-master ?

Reply:
that'd definitely alleviate the potential for confusion.  "owner: master" could work too.